cpmpy/solvers/minizinc.py

```diff
@@ -223,13 +223,34 @@ def _post_solve(self, mzn_result):
 
         # new status, translate runtime
         self.cpm_status = SolverStatus(self.name)
+        runtime = 0
         if 'time' in mzn_result.statistics:
             time = mzn_result.statistics['time']
             if isinstance(time, int):
                 self.cpm_status.runtime = time / 1000
             elif isinstance(time, timedelta):
                 self.cpm_status.runtime = time.total_seconds()  # --output-time
-
+        else:
+            if 'flatTime' in mzn_result.statistics:
```

Comment:
Is `mzn_result.statistics` just a dictionary? 
If so you can reduce this code somewhat using:
```python
runtime += mzn_time_to_seconds(mzn_result.statistics.get('flatTime',0))
```
The second argument of the `.get` method of a dictionary is a default value for it the key is not present